src/java/org/apache/cassandra/db/commitlog/CommitLogSegment.java

```diff
@@ -638,6 +638,7 @@ public String dirtyString()
         {
             TableMetadata m = Schema.instance.getTableMetadata(tableId);
             sb.append(m == null ? "<deleted>" : m.name).append(" (").append(tableId)
+              .append(", keyspace: ").append(m.keyspace)
```

Comment:
I assume this isn't JDK21 specific but something you noticed along the way @jmckenzie-dev